src/rpc/RPCEngine.h

```diff
@@ -149,24 +156,23 @@ class RPCEngineBase
             if (v)
                 return v->as_object();
             else
+            {
+                notifyErrored(ctx.method);
```

Comment:
One thing, do we need to change the name to logicError ? suggest it is not a real error

Reply:
We could change this `notifyErrored` and inside Counters but we can't change the json we generate in the report i think. So i left it as it was to match the key in the report itself.